src/sentry/models/integration.py

```diff
@@ -47,7 +48,7 @@ class Meta:
         unique_together = (("organization_integration_id", "external_id"),)
 
 
-class OrganizationIntegration(Model):
+class OrganizationIntegration(DefaultFieldsModel):
```

Comment:
@MeredithAnya I think you should be able to remove the `date_added` fields that are explicitly defined in this model, as well as the others.

Reply:
Hmm so I think this is probably okay for tables where `date_added` already has `null=True` but I'm not sure we need to make the other columns nullable if they don't have it already.

Reply:
@MeredithAnya Maybe we don't need to do that but I think part of the reason for this PR is to have as much consistency between the models as possible by letting the `DefaultFieldsModel` control the date fields.